### ISSUE TITLE
Nogs Level: First impression.

### ISSUE THREAD

**Issue by JamieRCHI** [Nogg-aholic/NogsLevel#1]
Found that the config option to skip the tutorial does not work. You still have to select this in the game options before starting a new game.

Tried the water level first. Had all the basic recipes I needed to start with the Cheats disabled.
Tried to place a Miner on one of the mineral nodes before placing foundations. Was able to place on the Caterium node but not a few others I tried. Probably not really a bug.

Then created a level with ground enabled.  The spawned nodes were in a different order. Would be nice to have the option for them to spawn in the order needed to start a new game. 
Mostly Iron Ore first, Copper second, and stone third, with Uranium last and with some space so I don't have to deal with the radiation issues.

Also found when first creating the level with ground,  that several recipes I needed, such as the one to make concrete were not available.  And I had no radiation effects from the Uranium node.
Saving the game and restarting fixed all that.

Would it be possible to create water tiles that would fit between foundations so I could place water where I want? There is a mod in Fallout 4 that does this.

I did see the big square that seems to be water, but it doesn't really look like water. (The water in the 'Water World' mode looked fine.) 

And in the future, would it be possible to create land using an imported file from the World Machine?

Thanks for this mod.

**Comment by budak7273** [comment]
Hey, thanks for the feedback.

This mod is mostly intended as an example world for other modders that want to make their own custom levels, and not so much a playable experience, but I'm trying my best to keep it playable as well. I appreciate you testing stuff.

- It looks like Nog set the nodes to shuffle order. As of 1.0.0, the vanilla resources should spawn in a consistent order, but the modded ones may not be due to how assets get loaded.
- The skip tutorial option should fully work now and grant starting recipes
- Uranium node radioactivity not kicking in is a bug with the base game
- You can build your own water with this mod: https://ficsit.app/mod/WaterPool
- That big square is water, but it bugged out in one of the past updates, should be fixed in 1.0.0
- Custom landmasses are something you would have to make your own mod to implement

If you have any other feedback, feel free to re-open this or open another issue item.
